validators/denali/versions-denali.md

```diff
@@ -0,0 +1,100 @@
+
+3.  Please, don't forget to check your SSL certificate using `skale ssl check` command.
+
+### Mainnet Versions for Set up
+
+**Node CLI**: [1.1.1](https://github.com/skalenetwork/node-cli/releases/download/1.1.1/skale-1.1.1-Linux-x86_64) 
+
+**Validator CLI**:  [1.2.1](https://github.com/skalenetwork/validator-cli/releases/tag/1.2.1)
+
+**SGX Wallet**: [1.70.0-stable.0](https://github.com/skalenetwork/sgxwallet/releases/tag/1.70.0-stable.0) Container: `skalenetwork/sgxwallet_release:1.70.0-stable.0`
+
+**Skale node**: [1.2.3](https://github.com/skalenetwork/skale-node/releases/tag/1.2.3)
+
+**Skaled**: [3.5.14-stable.0](https://github.com/skalenetwork/skaled/releases/tag/3.5.14-stable.0)
+
+**Docker lvmpy**: [1.0.1-stable.2](https://github.com/skalenetwork/docker-lvmpy/releases/tag/1.0.1-stable.2)
+
+**Transaction manager**: [1.1.0-stable.1](https://github.com/skalenetwork/transaction-manager/releases/tag/1.1.0-stable.1)
+
+**Skale admin**: [1.1.0-stable.2](https://github.com/skalenetwork/skale-admin/releases/tag/1.1.0-stable.2)
+
+**Bounty agent**: [1.1.1-stable.1](https://github.com/skalenetwork/bounty-agent/releases/tag/1.1.1-stable.1)
+
+**Skale watchdog**: [1.1.3-stable.1](https://github.com/skalenetwork/skale-watchdog/releases/tag/1.1.3-stable.1)
+
+### Mainnet Versions for Review
+
+**Node CLI**: [1.1.0](https://github.com/skalenetwork/node-cli/releases/download/1.1.0/skale-1.1.0-Linux-x86_64) 
+
+**Validator CLI**:  [1.2.0](https://github.com/skalenetwork/validator-cli/releases/tag/1.2.0)
+
+**SGX Wallet**: [1.70.0-stable.0](https://github.com/skalenetwork/sgxwallet/releases/tag/1.70.0-stable.0) Container: `skalenetwork/sgxwallet_release:1.70.0-stable.0`
+
+**Skale node**: [1.2.2](https://github.com/skalenetwork/skale-node/releases/tag/1.2.2)
+
+**Skaled**: [3.5.12-stable.1](https://github.com/skalenetwork/skaled/releases/tag/3.5.12-stable.1)
+
+**Docker lvmpy**: [1.0.1-stable.1](https://github.com/skalenetwork/docker-lvmpy/releases/tag/1.0.1-stable.1)
+
+**Transaction manager**: [1.1.0-stable.0](https://github.com/skalenetwork/transaction-manager/releases/tag/1.1.0-stable.0)
+
+**Skale admin**: [1.1.0-stable.1](https://github.com/skalenetwork/skale-admin/releases/tag/1.1.0-stable.1)
+
+**Bounty agent**: [1.1.1-stable.0](https://github.com/skalenetwork/bounty-agent/releases/tag/1.1.1-stable.0)
+
+**Skale watchdog**: [1.1.3-stable.0](https://github.com/skalenetwork/skale-watchdog/releases/tag/1.1.3-stable.0)
+
+#### Mainnet Environment Variables to set
+
+```shell
+MONITORING_CONTAINERS=True
+DOCKER_LVMPY_STREAM=1.0.1-stable.2
+MANAGER_CONTRACTS_ABI_URL=https://raw.githubusercontent.com/skalenetwork/skale-network/master/releases/mainnet/skale-manager/1.8.1/skale-manager-1.8.1-mainnet-abi.json
+IMA_CONTRACTS_ABI_URL=https://raw.githubusercontent.com/skalenetwork/skale-network/master/releases/mainnet/IMA/1.0.0/abi.json
```

Comment:
what about IMA? Is this one the correct one?

Reply:
This one is correct.